<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4700" started-at="2019-02-04T10:12:25Z" finished-at="2019-02-04T10:12:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4700" started-at="2019-02-04T10:12:25Z" finished-at="2019-02-04T10:12:30Z">
      <class name="com.qa.tester.sikuli_garcia">
        <test-method status="FAIL" signature="tests()[pri:0, instance:com.qa.tester.sikuli_garcia@3632be31]" name="tests" duration-ms="4685" started-at="2019-02-04T10:12:25Z" finished-at="2019-02-04T10:12:30Z">
          <exception class="org.sikuli.script.FindFailed">
            <message>
              <![CDATA[can not find P(C:/Users/Suvneet.Singh/Desktop/sikuli_ss/drop.PNG) S: 0.7 in S(0)[0,0 1440x900]]]>
            </message>
            <full-stacktrace>
              <![CDATA[FindFailed: can not find P(C:/Users/Suvneet.Singh/Desktop/sikuli_ss/drop.PNG) S: 0.7 in S(0)[0,0 1440x900]
  Line 2189, in file Region.java

at org.sikuli.script.Region.handleFindFailedShowDialog(Region.java:2189)
at org.sikuli.script.Region.handleFindFailed(Region.java:2134)
at org.sikuli.script.Region.wait(Region.java:2546)
at com.qa.tester.sikuli_garcia.tests(sikuli_garcia.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.sikuli.script.FindFailed -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tests -->
      </class> <!-- com.qa.tester.sikuli_garcia -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
